log/clog.go

```diff
@@ -0,0 +1,15 @@
+package log
+
+import (
+	"github.com/armory-io/go-commons/server"
+	"github.com/armory-io/go-commons/temporal"
+	"go.uber.org/zap"
+)
+
+func NewClog(ctx temporal.LoggingValuer) *zap.SugaredLogger {
+	v, ok := ctx.Value(server.RequestDetailsKey{}).(server.RequestDetails)
```

Comment:
you could use here server.ExtractRequestDetailsFromContext utility method - no need to do that manually.

Reply:
and - does that mean we should obsolete the other ways of creating loggers and use only NewClog since now on?

Reply:
For the first comment, I already fix that.

Reply:
For the second comment, yes, the goal of this PR is to introduce a new way to instantiate a logger within the application, the logger will have enough `context` information that will help us track request, workflows, activities using trace.id, span.id, etc.

You will be able to add any extra contextual information to the logger.